server.js

```diff
@@ -710,10 +710,24 @@ module.exports.initExpress = function() {
         require('./pages/studentAssessmentInstanceHomework/studentAssessmentInstanceHomework'),
         require('./pages/studentAssessmentInstanceExam/studentAssessmentInstanceExam'),
     ]);
+    app.use('/pl/course_instance/:course_instance_id/instructor/assessment/:assessment_id/assessment_question/:assessment_question_id/unlock_ungraded', [
+        function(req, res, next) {res.locals.assessment_question_id = req.params.assessment_question_id; next();},
+        require('./pages/instructorQuestionManualGrading/instructorQuestionManualGradingUnlockInstanceQuestions'),
```

Comment:
We are using a convention of one route handler per directory in `pages/`. That is, `instructorQuestionManualGrading.js` should be the only handler in `pages/instructorQuestionManualGrading/`. We should either handle this query within `instructorQuestionManualGrading.js` or add another directory in `pages`.

Reply:
This endpoint was eliminated, but I implemented convention this for the next_ungraded endpoint.